js/templates/account.html

```diff
@@ -36,6 +36,8 @@
 <div class="folders"></div>
 {{#unless isUnifiedInbox}}
 {{#if hasFolders}}
-<li class="account-toggle-collapse">{{toggleCollapseMessage}}</li>
+<li class="account-toggle-collapse">
+	<a href="">{{toggleCollapseMessage}}</a>
```

Comment:
here you go. The newly added `a` element causes an unexpected navigation. The click handler should call `event.preventDefault()`.

Reply:
preventDefault isn't working here, wefound something else. ;)